lessons/Condition_continued.md

```diff
@@ -70,3 +70,61 @@ else:
   print("deactivate shade mode")
   
 ```
+homework assignment
+```py
+guestName="Akshita"
+guestRoomNo=502
+
+# if guestName.title()== "Akshita" and guestRoomNo == 502:
+#    print("allow access to guest")
+# else:
+#     print("no access allowed")
+#     if guestName.title() > "akshita" and guestRoomNo > or < 502:
+#       print("allow acess to guest")
+
+
+salaryHigh=100000,90000,98000,87000,82000
```

Comment:
@sainidevika12 - please see that you are using square brackets for proper list usage.